### ISSUE TITLE
Tiny Issues

### ISSUE THREAD

**Issue by TotoR115** [Gibberlings3/BG1_Romantic_Encounters#17]
- TICAMBAG.bam is not centered
- TICAMBAG.STO can store Miscellaneous (0) items witch is weird for a book bag

**Comment by Gitjas** [comment]
Thanks!
That the bag can store Miscellaneous items is due to the fact that Camryn's books are tagged as one so they can be put into the quickslot to retrieve the letter out of it. It would be weird not to be able to put these into the bag. Also, I noticed that some books are tagged as Miscellansous themselves, not "books" - I forgot whether it was for BGT or BG:EE.
(That it's possible to put the corpses from the sewers inside is probably weirder, but... I don't have a good solution for this and went with the "I want all books to go inside the bag".)

**Comment by TotoR115** [comment]
Hi Jastey,
I understand the fist goal but Miscellaneous items are not so rare and could transform this Book case into back of holding corpses (disgusting...). 

If I may propose, those books could be tag as `"food"` or `"keys".` both seems better and will reduce the weirdness... who do not put his keys or his lunch box in his satchel ?

> Also, I noticed that some books are tagged as Miscellaneous themselves, not "books" - I forgot whether it was for BGT or BG:EE.


In BG2/BGT (and maybe EET, *it need to be confirmed*) all regular books are tag as `books`. In BG1 and Tutu books are tags as `Miscellaneous`.*For BGEE, I don't know, it has to be checked*.

So I would say, that there is not one solution here, as one will work better with tutu and one with bgt... Maybe it could be possible to create 2 sore (TICAMBAG.STO) that are installed with conditions : BGT/EET (food or keys) or Tutu/*BGEE (to be confirmed)* and to mod books from this mod in accordance

**Comment by Gitjas** [comment]
I had a look now. In BG:EE *all* books are tagged as Miscellaneous, not only the ones that need to be used in the quickslot. BGT and EET tagg most normal books as "books".  But all tomes, manuals, other books that can be put into the quickslot (Book of Infinite Spells e.g.), but also a handful of normal books are tagged as Miscellaneous.
So, the "Miscellaneous" tagg will stay I'm afraid. I don't see a use in a book bag where some (or, for BG:EE all) books do not go in. 
At some point, players have to decide how they want to play the game. If they want to take advantage of engine limitations (because that's what this is) and put corpses into the book bag together with the Tome of Clear Thought  - it's their decision.
In other words: I have no perfect solution for this, so I go with this one.

**Comment by Gitjas** [comment]
I'm not ignoring the BAM report, but I seem to be unable to improve the centering in EE. If anyone wants to take a look at that, please do.
The second Frame is centered and looks good for BGT.

**Comment by Gitjas** [comment]
Turns out that was pretty easy to change. I misunderstood what you meant and should have asked earlier.
BAM is fixed and the change will be in the next version.
Thanks!

**Comment by Gitjas** [comment]
`TICAMBAG.bam is not centered`
should be fixed in v11.
Closing this. Thanks again!
